includes/class-wc-stripe-blocks-support.php

```diff
@@ -81,25 +81,27 @@ public function get_payment_method_script_handles() {
 	 * @return array
 	 */
 	public function get_payment_method_data() {
+		$payment_request_configuration = new WC_Stripe_Payment_Request();
```

Comment:
Are there any downsides in initializing `WC_Stripe_Payment_Request` here too? Like adding actions/filters twice.

Edit: I added a log to the construct and it looks like this class is being instantiated several times actually 😅

Reply:
This will be handled once https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1585 is merged so I didn't address it here.

I should've mentioned it here though, sorry about that!

Reply:
> class is being instantiated several times actually 🤔

Reply:
> I'm not sure where else we're instantiating the class

Well I ran some more tests and apparently when you _refresh_ the **WooCommerce > Status > Logs** page it ends up logging 5 times whatever is in the construct of `WC_Stripe_Payment_Request` 🤷🏻‍♂️. Probably due to multiple redirects + the fact that `WC_Stripe` is initialized with WooCommerce.

When loading the cart page it seems to be initializing it correctly, but twice on the block cart due to the line I commented above.

Since this will be handled in #1585. I'm marking this as resolved 👍